### ISSUE TITLE
iOS Widgets

### ISSUE THREAD

**Issue by gbu117** [holzschu/a-shell#166]
It would be awesome to be able to have formatted widgets with the output of commands which could be placed on the home screen and/or dashboard, which combined with something similar to the watch command could allow live refreshing of the output. I am not sure how iOS widgets work, so if this isn't possible, I understand!

**Comment by holzschu** [comment]
Hi, 
Yes, it's on the list of things to do, but there are lots of other things on that list. I will get there eventually.

**Comment by gbu117** [comment]
Of course, sorry I did not mean to seem rude, I just wanted to put it out there! Thank you for all the hard work you put in!

**Comment by TheGamer1002** [comment]
Bump! This’d make a great addition.
